app.js

```diff
@@ -49,11 +49,22 @@ Test this function by hand in the console to get it working, and when you think
 
 // Write your code here
 function sumAndMultiply(a,b,c){ //eslint-disable-line
-
+  var sumAndMultiplyValues = [];
+  sumAndMultiplyValues[0] =
+  sum(a,
+    sum(b,c)[0]
+  )[0];
+  sumAndMultiplyValues[1] =
+  multiply(a,
+    multiply(b,c)[0]
+  )[0];
+  sumAndMultiplyValues[2] = a + ' and ' + b + ' and ' + c + ' sum to ' + sumAndMultiplyValues[0] + '.';
+  sumAndMultiplyValues[3] = 'The product of ' + a + ' and ' + b + ' and ' + c + ' is ' + sumAndMultiplyValues[1] + '.';
+  console.log(sumAndMultiplyValues);
 }
 
 // Here is the test for sumAndMultiply(); uncomment it to run it
-// testSumAndMultiply(4,7,5);
+sumAndMultiply(4,7,5);
```

Comment:
You should not need to change the code here, only uncomment. Looks like `tests.js` is not included in your `index.html` before `app.js` is loaded?